docs/src/pages/reference/builtin-components.md

```diff
@@ -5,6 +5,26 @@ title: Built-In Components
 
 Astro includes several builtin components for you to use in your projects. All builtin components are available via `import {} from 'astro/components';`.
 
+
+## `<Code />`
+
+```astro
+---
```

Comment:
It'd be really cool to make use of https://github.com/unlight/remark-sources so that we can provide the rendered version of this code example inline with the component!